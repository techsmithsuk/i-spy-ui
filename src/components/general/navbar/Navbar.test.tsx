import React from 'react';
import { Navbar } from './Navbar';
import renderer from 'react-test-renderer';
import { BrowserRouter} from "react-router-dom";
import ReactDOM from 'react-dom';

describe("Navbar", () => {
    it("Should render without errors", () => {
        const component = renderer.create(
            <BrowserRouter>
            <Navbar/>
            </BrowserRouter>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
