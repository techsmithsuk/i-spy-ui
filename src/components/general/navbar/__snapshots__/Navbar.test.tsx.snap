// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Should render without errors 1`] = `
<nav>
  <div
    className="main-nav-elements"
  >
    <div>
      <a
        className="nav-element"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </div>
    <div>
      <a
        className="nav-element"
        href="/reports"
        onClick={[Function]}
      >
        Reports
      </a>
    </div>
    <div
      className="super-admin"
    >
      <a
        className="nav-element"
        href="/users"
        onClick={[Function]}
      >
        Manage Users
      </a>
    </div>
    <div
      className="super-admin"
    >
      <a
        className="nav-element"
        href="/settings"
        onClick={[Function]}
      >
        Admin Settings
      </a>
    </div>
  </div>
  <div
    className="logOut"
  >
    <div>
      <a
        className="nav-element"
        href="/login"
        onClick={[Function]}
      >
        Log Out
      </a>
    </div>
  </div>
</nav>
`;
